modules/api1/docs.md

```diff
@@ -57,6 +57,32 @@ You may pass additional parameters when querying a list:
   returned.
 - `yiiVersion` - version of the framework project built with. Either `1.0` or `1.1`.
 
+
+### Get primary image <a href="#projects-view-primary-image" id="projects-view-primary-image">#</a>
```

Comment:
Primary image ID by itself isn't useful. I'd include this info into `GET /project` if it's interesting to expose it via API at all...